Dockerfile

```diff
@@ -1,43 +1,62 @@
 # Use a specific Node.js version for better reproducibility
 FROM node:23.3.0-slim AS builder
 
-# Install pnpm globally and install necessary build tools
+# Install pnpm globally and necessary build tools
 RUN npm install -g pnpm@9.4.0 && \
     apt-get update && \
-    apt-get install -y git python3 make g++ && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        git \
+        python3 \
+        python3-pip \
+        curl \
+        node-gyp \
+        ffmpeg \
+        libtool-bin \
+        autoconf \
+        automake \
+        libopus-dev \
+        make \
+        g++ \
+        build-essential \
+        libcairo2-dev \
+        libjpeg-dev \
+        libpango1.0-dev \
+        libgif-dev \
+        openssl \
+        libssl-dev && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
 # Set Python 3 as the default python
-RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN ln -sf /usr/bin/python3 /usr/bin/python
 
 # Set the working directory
 WORKDIR /app
 
-# Copy package.json and other configuration files
-COPY package.json pnpm-lock.yaml pnpm-workspace.yaml .npmrc turbo.json ./
+# Copy application code
+COPY . .
 
```

Comment:
Not sure if copying all files is necessary since it floats the image. Previously, including just the necessary files for building the image is appropriate.

Id only suggest including the `client` folder & `lerna.json` since those are included for the build now.